lib/copy_peste/command/init_bdd.rb

```diff
@@ -13,9 +13,9 @@ def run()
         config_path = File.join(Require::Path.root, './', 'copy_peste.yml')
         config_path = File.expand_path config_path
         config = YAML::load_file(config_path)
-        if File.directory? config['port_tree_path']
-          @graph_com.display(10, "Loading directory #{config['port_tree_path']}")
-          scan = ScanSystem.new(config['port_tree_path'])
+        if File.directory? config['ports_tree_path']
```

Comment:
This isn't "just a string" that is used here and there, but a key to config value. Totally legitimate to keep it outside a variable.

However, there's a problem above I just spotted. Key was originally singular so I changed it to plural but I missed one :facepalm: